src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -38,204 +38,230 @@
   <ValueType Name="InterpolatedStringHandlerData"/>
   <ValueType Name="WellKnownMember"/>
   <ValueType Name="CollectionExpressionTypeKind"/>
-
```

Comment:
My suggestion would be to manually go through this diff and revert the unnecessary changes like these blank line removals. I understand that it's a bit of work to do, but I think that it will produce a better diff and that the XML-first approach is better for the long-term health of the code.